include/envoy/http/header_map.h

```diff
@@ -749,6 +749,8 @@ class RequestHeaderMap
 };
 using RequestHeaderMapPtr = std::unique_ptr<RequestHeaderMap>;
 using RequestHeaderMapOptRef = absl::optional<std::reference_wrapper<RequestHeaderMap>>;
+using RequestHeaderMapOptConstRef = absl::optional<std::reference_wrapper<const RequestHeaderMap>>;
+using RequestHeaderMapOptRef = absl::optional<std::reference_wrapper<RequestHeaderMap>>;
```

Comment:
nit: Lines 751 and 753 seem to be providing double definitions for RequestHeaderMapOptRef